python/rmm/rmm.py

```diff
@@ -32,7 +32,6 @@ def reinitialize(
     pool_allocator=False,
     managed_memory=False,
     initial_pool_size=None,
-    devices=0,
```

Comment:
Why are we removing the `devices` kwarg here? Is RMM no longer able to handle multiple devices in a single process?

Reply:
Cnmem was the only thing that handled multiple devices. This is the new, more generic solution: https://github.com/rapidsai/rmm/pull/458

Reply:
Does that mean all other resources will use device `0`? Or should we be enforcing that somehow if we want that?

Reply:
All resources (except cnmem) use whatever the current device is, i.e., whatever is returned from `cudaGetDevice`.

Reply:
Does this work for Dask-CUDA @pentschev? I'm guessing so, but want to double check.

Reply:
Dask-CUDA just shuffles the `CUDA_VISIBLE_DEVICES` environment variable so as to avoid needing to call `cudaSetDevice` / `cudaGetDevice` as there's libraries with legacy code that explicitly calls `cudaSetDevice(0)` within them.

Reply:
Thanks @jrhemstad for the answer!

Reply:
Yes, it works for dask-cuda. To be more specific, dask-cuda sets `CUDA_VISIBLE_DEVICES` in a round-robin fashion for each process and the active device is always the first in `CUDA_VISIBLE_DEVICES` list.

Reply:
Sorry @kkraus14 I didn't mean to correct you, it was just that GitHub didn't refresh comments while I wrote this.

Reply:
Thanks for confirming 😄

Reply:
Note we will want to expose #458 in Python as well.